src/sparql/SyntaxBuilder.ts

```diff
@@ -1,7 +1,7 @@
 import {
 	BgpPattern,
-	BlankTerm,
-	IriTerm,
+	BlankTerm, FilterPattern,
```

Comment:
nitpick: Can all be either one per line or filled as much as possible not somewhere in between?

Reply:
No idea why my IDE didn't do that by itself. Fixed.